packages/nimbuild-corejs/lib/tests/__snapshots__/index.test.js.snap

```diff
@@ -760,3 +881,5 @@ Array [
   "intersection-observer",
 ]
 `;
+
+exports[`index.js proactively handles future versions of chrome family by assuming latest supported 1`] = `Array []`;
```

Comment:
Empty array here means no polyfills are being generated, which should be the expected behavior for modern browsers.